idl/partition_checksum.idl.hh

```diff
@@ -129,8 +134,10 @@ struct node_ops_cmd_request {
     std::list<gms::inet_address> leaving_nodes;
     // Map existing nodes to replacing nodes
     std::unordered_map<gms::inet_address, gms::inet_address> replace_nodes;
+    std::unordered_map<gms::inet_address, std::list<dht::token>> bootstrap_nodes;
```

Comment:
So, different verbs will have different fields set in this structure?

Please add documentation.

Reply:
Yes, documentation is added.